rdr_service/resource/calculators/participant_ubr.py

```diff
@@ -132,13 +131,14 @@ def ubr_ethnicity(answers):
         """
         if answers is None:
             return UBRValueEnum.NotAnswer_Skip
+
         # Note: assume UBR by default and check for exclusion criteria.
         answers = answers.split(',')
         if len(answers) == 1:
```

Comment:
Do we need to confirm there is no unexpected/non-conformant response data ingested by the RDR where there are multiple answer codes but one of them is `PMI_Skip` or `PMI_PreferNotToAnswer`?

Reply:
I looked at the possible answers for 'Race_WhatRaceEthnicity' and there are no cases where a Skip or Prefer Not To Answer are mixed with other answers.  I think this logic is fine in this method.